modules/http-load-balancer-website/variables.tf

```
@@ -53,6 +53,12 @@ variable "ssl_certificate" {
   default     = ""
 }
 
+variable "ssl_policy" {
+  description = "Link to ssl policy. Omitted if not provided"
+  type        = string
+  default     = ""
```

```suggestion
  default     = null
```
Let's set the default to `null` for better consistency with our other modules.